objects/src/accounts/data.rs

```diff
@@ -85,9 +85,11 @@ impl Deserializable for AccountData {
 // ================================================================================================
 
 /// AuthData is a representation of the AuthScheme struct meant to be used
-/// for Account serialisation and deserialisation for transport of Account data
+/// for Account serialisation and deserialisation for transport of [AccountData]
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub enum AuthData {
+    /// RNG Seed for generating the RPO Falcon secret key, expected to be generated
+    /// with [miden_crypto::rand::RpoRandomCoin]
     RpoFalcon512Seed([u8; 32]),
```

Comment:
Looking at this now, I'm wondering if we should put the actual private key in here. This way, we won't need to make any assumptions about how the key s generated from the seed. The only drawback that I see is that the key is quite a bit bigger (i.e., ~1KB) - but I'm not sure if that's a problem here.

Reply:
Yeah, I definitely thought it was discarded as an option originally and so I did not consider it for this change, but that sounds better.

Reply:
In this case, I wonder if this can be replaced by [AuthSecretKey](https://github.com/0xPolygonMiden/miden-base/blob/next/miden-tx/src/host/tx_authenticator.rs#L51C10-L51C23) struct (we'd move it to `miden-objects`). Seems like they'd be basically identical.